schemas/src/registry_schemas/schemas/annual_report.json

```diff
@@ -13,7 +13,8 @@
             "required": [
                 "annualGeneralMeetingDate",
                 "annualReportDate",
-                "directors"
+                "directors",
+                "offices"
```

Comment:
This is a good change, but will require a change to the format that COOPS Annual Reports are submitted. They currently just submit delivery and mailing without a Wrapper, but i can/will update that in the Annual Report PR currently up.